src/helix-ui/styles/components/navigation.less

```diff
@@ -53,7 +53,7 @@
     text-transform: uppercase;
   }
 
-  a {
+  a[href] {
```

Comment:
Now that link styles are defined before this file, these selectors no longer matched and needed updated.